pkg/installer/README.md

```diff
@@ -30,3 +30,39 @@ In addition, it expects to have the following:
 * `mkimage-raw-efi` image available read-only at `/mkimage`. This is set in the [installer.yml](../../images/installer.yml.in) file. It uses it to launch `make-raw`, in order to write to the target installation disk.
 * `/bits/rootfs.img` - the rootfs image to install. This is set in the [installer.yml](../../images/installer.yml.in) file.
 * `/bits/persist.img` - the persist image to install. This is set in the [installer.yml](../../images/installer.yml.in) file.
+
+## Installer interactive mode
+
+### Description
+
+The interactive mode of installer provides a user-friendly text-based user interface (TUI) to configure the parameters of the installation. Utilizing a TUI, allows users to configure installation parameters with ease. The primary goal of this installer is to simplify the installation process on edge devices while offering flexibility and automation, similar to the installation wizards used for setting up new Wi-Fi routers.
+
+### Configuration steps
+
+Users can configure the installation parameters through a series of steps, with the ability to navigate back and forth until their choices are finalized. The steps are as follows:
+
+1. Choose Filesystem Type: Select from options such as squashfs, EXT3, EXT4, and ZFS.
```

Comment:
We don't support squashfs for /persist AFAIK.

Reply:
And your code does not allow squashfs, so only issue is the README reference to squashfs.

Reply:
I updated the configuration steps in README.md to match our current procedure.